test.ts

```diff
@@ -126,30 +170,37 @@ test('maxAge items are deleted even if function throws', async t => {
 
 	const cache = new Map();
 	const memoized = mem(fixture, {maxAge: 100, cache});
+	// @ts-expect-error
 	t.is(memoized(1), 0);
+	// @ts-expect-error
 	t.is(memoized(1), 0);
 	t.is(cache.size, 1);
 	await delay(50);
+	// @ts-expect-error
 	t.is(memoized(1), 0);
 	await delay(200);
-	t.throws(() => memoized(1), {
-		message: 'failure'
-	});
+	// @ts-expect-error
+	t.throws(() => memoized(1), {message: 'failure'});
 	t.is(cache.size, 0);
 });
 
 test('cache option', t => {
 	let i = 0;
 	const fixture = () => i++;
 	const memoized = mem(fixture, {
+		// @ts-expect-error
```

Comment:
This should not be ignored though. The `cache` option should support WeakMap.

Reply:
That's right. However #66 is already touching those exact types so perhaps it's best to merge this and have that PR fix any remaining issues.

If you prefer, I can drop this comment and have it fail for the PR to pick up.

Reply:
```suggestion
```